decidim-proposals/app/views/decidim/proposals/admin/proposals/show.html.erb

```diff
@@ -49,80 +49,25 @@
             </p>
           </div>
         </div>
+
         <% presented_endorsers = endorsers_presenters_for(proposal).first(6).to_a %>
```

Comment:
This suggestion i previously made, is actually introducing a bug (ex seeing admin 6 endorsers, but in "endorsement_count" 8, in public web 8). Removing `.to_a` solves the issue. 

```suggestion
        <% presented_endorsers = endorsers_presenters_for(proposal) %>
```